R/estimate_grouplevel.R

```diff
@@ -69,12 +69,20 @@ estimate_grouplevel <- function(model, type = "random", dispersion = TRUE, test
     ...
   )
 
+  # get cleaned parameter names with additional information
+  clean_parameters <- attributes(params)$clean_parameters$Group
+
+  # fix for rstanarm, which contains a sigma columns
+  if (inherits(model, "stanreg")) {
+    clean_parameters <- clean_parameters[clean_parameters$Component != "sigma", ]
```

Comment:
In theory there can be models where you have random effects on `sigma` à-la-brms, but I quickly checked rstanarm's functions and I don't think there is currently any model implemented that would allow that.

However, this might change in the future so we might want to make a note of it or think of a more robust solution